packages/surveys/src/components/questions/matrix-question.tsx

```
@@ -133,14 +135,14 @@ export const MatrixQuestion = ({
               <thead>
                 <tr>
-                  <th className="fb-px-4 fb-py-2"></th>
+                  <th className="fb-px-4 fb-py-2" />
                   {columnsHeaders}
                 </tr>
               </thead>
               <tbody>
                 {questionRows.map((row, rowIndex) => (
                   // Table rows
-                  <tr className={`${rowIndex % 2 === 0 ? "bg-input-bg" : ""}`}>
+                  <tr className={rowIndex % 2 === 0 ? "bg-input-bg" : ""}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)